public/app/core/utils/explore.ts

```diff
@@ -94,7 +94,7 @@ export async function getExploreUrl(args: GetExploreUrlArguments): Promise<strin
   const exploreState = JSON.stringify({
     [generateExploreId()]: { range: toURLRange(timeRange.raw), queries: interpolatedQueries, datasource: dsRef?.uid },
   });
-  return urlUtil.renderUrl('/explore', { panes: exploreState, schemaVersion: 1 });
+  return urlUtil.renderUrl(`${config.appSubUrl}/explore`, { panes: exploreState, schemaVersion: 1 });
```

Comment:
We have a util for this already , ensureBase or something like that

Reply:
Interesting, will have a look

Reply:
Alright, changed to using `assureBaseUrl`